src/internal/version-check.js

```diff
@@ -7,4 +11,13 @@
   if (local !== JSON.parse(stable.text).version && local !== JSON.parse(exp.text).version) {
     global.logger.warn('Not up-to-date with any remote version, update recommended')
   }
-})()
+})
+
+async function gitVerify () {
+  const currentHead = await exec('git rev-parse --abbrev-ref HEAD')
+  const remoteRef = await exec(`git rev-parse origin/${currentHead.stdout.trim()}`)
+  const currentRef = await exec('git rev-parse HEAD')
+  const count = await exec(`git rev-list ${currentRef.stdout.trim()}..${remoteRef.stdout.trim()} --count`)
+  if (count.stdout.trim() > 0) global.logger.warn(`You're behind ${count.stdout.trim()} commit(s) compared to remote branch ${currentHead.stdout.trim()}, update recommended`)
+  else global.logger.log('Fully up-to-date, nice!')
```

Comment:
Language revision: I suggest using **You're fully up to date!** instead.

Reply:
Dunno, I like the less serious tone of the current message more.